src/views/createEditCommunity/__tests__/create.test.tsx

```diff
@@ -1129,9 +1129,11 @@ describe('create community', () => {
                 queryByText(i18n.t('createCommunity.communityRequestSending'));
             }, 5000)
         ).not.toBeNull();
-        expect(
-            queryByText(i18n.t('createCommunity.communityRequestSuccess'))
-        ).toBeNull();
+        setTimeout(() => {
```

Comment:
No `setTimeout`.
Use different query.
https://testing-library.com/docs/queries/about/#types-of-queries

Reply:
Fixed.